core/dev/fhir/fhir-deployment.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: fhir
+  name: fhir
```

Comment:
@bradsawadye Please change the name here to match the naming convention. ie: `fhir-deployment`